### PR TITLE
Var Scope Fix + Maven Script Development

### PR DESCRIPTION
**Pull request by alexjhawk**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Ewon Flexy Extensions Library Changelog
 
-## Version 1.13.5
+## Version 1.13.6
```
Comment (TomKimsey) [inline CHANGELOG.md:3]: Was skipping 1.13.5 intentional?

Comment (alexjhawk) [reply CHANGELOG.md]: Good catch. 1.13.5 was not skipped, although it did fail to deploy to the Maven repo. 

This should've been a separate changelog for 1.13.6.

Comment (alexjhawk) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Ewon Flexy Extensions Library Changelog
 
+## Version 1.13.6
+### Features
+- N/A
+### Bug Fixes
+- Fixed an issue in ApplicationControlApiListener.java where potentially helpful constants were not properly scoped for access by subclasses.
```
Comment (TomKimsey) [inline CHANGELOG.md:7]:
```suggestion
- Fixed an issue in ApplicationControlApiListener.java where constants were not properly scoped for access by subclasses.
```

**Review by TomKimsey — APPROVED** [review]
Approved w/ suggestion

File: CHANGELOG.md (hunk 2 above)
Comment (alexjhawk) [reply CHANGELOG.md]: Implemented your exact suggestion.

**Review by oliver-walker-hms — APPROVED** [review]

**Pull request merged**